api/app/src/command/medical/facility/update-facility.ts

```diff
@@ -0,0 +1,26 @@
+import BadRequestError from "../../../errors/bad-request";
+import { Facility } from "../../../models/medical/facility";
+
+export const updateFacility = async ({
+  id,
+  organizationId,
+  cxId,
+  data,
+}: {
+  id: string;
+  organizationId: number;
+  cxId: string;
+  data: object;
+}): Promise<Facility> => {
+  const [count, rows] = await Facility.update(
+    {
+      data,
+    },
+    { where: { id, organizationId, cxId }, returning: true }
+  );
+  if (count != 1)
```

Comment:
It can equal to zero if no record was found, right?

Reply:
Good point - will update the error accordingly